### PR TITLE
Prime cache macos

### PR DESCRIPTION
**Pull request by wli3**
Prime cache macOS with telemetry notice in conclusion page

screen shot 2017-04-27 at 1 54 50 pm

apr-27-2017 14-09-00

apr-27-2017 14-08-55

#6429 #6370

### CODE REVIEW

**Comment by wli3** [comment]
@richlander for the text

**Comment by livarcocc** [comment]
@MattGertz for approval.

**Comment by livarcocc** [comment]
@dotnet-bot Test Debian8.2 x64 Debug Build

File: packaging/osx/clisdk/resources/cs.lproj/firstTimeWelcomeMessage.html [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+            Telemetry
+        </h3>
+         <p>
+            The .NET Core tools collect usage data in order to improve your experience. The data is anonymous and does not include command-line arguments. The data is collected by Microsoft and shared with the community. You can opt out of telemetry by setting a DOTNET_CLI_TELEMETRY_OPTOUT environment variable to 1 using your favorite shell. You can read more about .NET Core tools telemetry at <a href="https://aka.ms/dotnet-cli-telemetry">https://aka.ms/dotnet-cli-telemetry</a>.
+        </p>
+        <h3>
+            Configuration
+        </h3>
+        <p>
+            A command is run in the end of install process to initially populate your local package cache, to improve restore speed and enable offline access. This command will take up to a minute to complete and will only happen once.
```
Comment (eerhardt) [inline packaging/osx/clisdk/resources/cs.lproj/firstTimeWelcomeMessage.html]: This has already happened right?  Is this section necessary?

Do people who run the installer care what commands the installer executed?  We don't tell them about other things that the installer did, like copy files, etc.

Comment (wli3) [inline packaging/osx/clisdk/resources/cs.lproj/firstTimeWelcomeMessage.html]: It is odd, since I moved it to after installation. I am thinking to send a different pull request to fix and unify the text across. I am adding the deb version currently, and there might be more change to the text.

#6448
